<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CGame</class>
 <widget class="QWidget" name="CGame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1356</width>
    <height>816</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="CMyView" name="mapView">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>180</y>
     <width>1151</width>
     <height>621</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>151</width>
     <height>391</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="areaLayout">
    <item>
     <widget class="QLabel" name="areaLabel">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="text">
       <string>Areaübersicht</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignHCenter|Qt::AlignTop</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>151</width>
     <height>231</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="playerLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <item>
     <widget class="QLabel" name="playerLabel">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="text">
       <string>Spielerübersicht</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignHCenter|Qt::AlignTop</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>321</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="informationLayout">
    <item>
     <widget class="QLabel" name="actTurnLabel">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Aktuelle Runde: </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="actPlayerLabel">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Aktueller Spieler: </string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>70</y>
     <width>531</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="turnControlLayout">
    <item>
     <widget class="QPushButton" name="startTurnButton">
      <property name="text">
       <string>Zug Beginnen</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="endSetButton">
      <property name="text">
       <string>Setzen Beenden</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="endTurnButton">
      <property name="text">
       <string>Zug Beenden</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>120</y>
     <width>531</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="gameControlLayout">
    <item>
     <widget class="QPushButton" name="saveButton">
      <property name="text">
       <string>Speichern</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="endGameButton">
      <property name="text">
       <string>Partie Beenden</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="quitGameButton">
      <property name="text">
       <string>Spiel Verlassen</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="instructionLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>891</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Hinweis:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>70</y>
     <width>191</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="specialControlLayout">
    <item>
     <widget class="QPushButton" name="quitActionPB">
      <property name="text">
       <string>Aktion abbrechen</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="changeCardsPB">
      <property name="text">
       <string>Karten eintauschen</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="countryInitPB">
      <property name="text">
       <string>Länderauswahl beginnen</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CMyView</class>
   <extends>QGraphicsView</extends>
   <header>myview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
